### PR TITLE
 Handle both external and internal policies.

### PR DESCRIPTION
**Pull request by AlbertoPdRF**
A change is also needed on https://github.com/thewca/wca-documents for this to work!

@jonatanklosko could you review? :slightly_smiling_face:

### CODE REVIEW

File: WcaOnRails/app/views/panel/index.html.erb [hunk 1]
```diff
@@ -6,12 +6,7 @@
   <h2>Staff</h2>
 
   <ul>
-    <li>
-      <%= link_to "WCA Deceased Registered Speedcubers Policy", "https://drive.google.com/open?id=1IAz5XFWFIXuweTyKoh4nrgQQnw9BO6Su" %>
-    </li>
-    <li>
-      <%= link_to "WCA Delegate Probationary Status Policy", "https://drive.google.com/open?id=1p9cXihULfAy_QS7SFNrs07JEf3cBQMxr" %>
-    </li>
+    <%= documents_list("policies/internal") %>
```
Comment (jfly) [inline WcaOnRails/app/views/panel/index.html.erb]: I'm surprised you didn't have to add DocumentsHelper to the panel controller as well.

Comment (AlbertoPdRF) [reply WcaOnRails/app/views/panel/index.html.erb]: I actually forgot to add it, but weirdly enough it seems to work :confused: 

I'll add it anyway, thanks for the catch!

**Comment by jfly** [comment]
Let's coordinate over slack with when to merge this. @AlbertoPdRF , have you put together a corresponding PR for wca-documents yet?

**Comment by AlbertoPdRF** [comment]
Yes, it's here: https://github.com/thewca/wca-documents/pull/88

It still needs an approving review from someone else from @thewca/board though.

**Pull request merged**